apps/aechannel/src/aesc_limits.erl

```diff
@@ -0,0 +1,90 @@
+-module(aesc_limits).
+
+-behavior(gen_server).
+
+-export([ allow_new/0
+        , register_returning/0 ]).
+
+-export([ start_link/0
+        , init/1
+        , handle_call/3
+        , handle_cast/2
+        , handle_info/2
+        , terminate/2
+        , code_change/3
+        ]).
+
+-record(st, { }).
+
+-define(PIDS, aesc_limits_pids).
+
+allow_new() ->
+    gen_server:call(?MODULE, allow_new).
+
+register_returning() ->
+    gen_server:call(?MODULE, register_returning).
+
+start_link() ->
+    ets:new(?PIDS, [set, public, named_table]),
```

Comment:
Why public?

Reply:
Easier to debug in some corner cases, but they certainly don't have to be public.

Reply:
Actually, also because the ets table is created and owned by the supervisor.
In order to make it private, the table will need to be inherited by the `aesc_limits` server, which is of course doable.